sap-lama-apps/nestedtemplates/osprofile.json

```diff
@@ -0,0 +1,77 @@
+{
```

Comment:
this can be simplified a bit by removing the nested deployment to just create the var - see:  https://github.com/Azure/azure-quickstart-templates/blob/master/100-marketplace-sample/azuredeploy.json#L299